api/setup/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const mongoURL =
-'mongodb+srv://XXXX:XXXX@<sandbox.r05hnxv.mongodb.net/myFirstDatabase>'
-export const secret = 'myStrongSecretCode'
+export const mongoDatabaseURL =
+'mongodb+srv://XXXX:XXXX-basics@<XXXX/myFirstDatabase>'
+export const passportSecretCode = 'myStrongSecretCode'
```

Comment:
Im not sure if you understood what environmental variables are. Please, read this https://nodejs.dev/en/learn/how-to-read-environment-variables-from-nodejs/

You are treating environmental variables as constants and you should not do that. They are two different things

Reply:
@CharlieBrownCharacter I think, I confused you with my linked_in message by providing two different links. Because the changes I made after our conversation are in another link https://github.com/dhiddiSk/userProfile_nodeApi/compare/main...fixBrunoComments  

Yeah, I have declared those values as environmental variables :)